src/modules/apis.js

```diff
@@ -0,0 +1,52 @@
+import { calcWindDirection, getLocalTime, calcTimeDiffInMin } from "./utilities";
```

Comment:
I like the api calls being in a separate module, looks very clean in your App.js because of it